tests/integration/targets/zypper/tasks/zypper.yml

```diff
@@ -255,6 +255,91 @@
     that:
      - "stat_result.stat.exists == true"
 
+# Build and install an empty rpm with error in post script
+- name: uninstall post_error
+  zypper:
+    name: post_error
+    state: removed
+
+- name: install rpmbuild
+  zypper:
+    name: rpmbuild
+    state: present
+
+- name: clean zypper RPM cache
+  file: 
+    name: /var/cache/zypper/RPMS
+    state: absent
+
+- name: create directory
+  file:
+    path: "{{ remote_tmp_dir | expanduser }}/zypper2"
+    state: directory
+
+- name: copy spec file
+  copy:
+    src: post_error.spec
+    dest: "{{ remote_tmp_dir | expanduser }}/zypper2/post_error.spec"
+
+- name: build rpm
+  command: |
+    rpmbuild -bb \
+    --define "_topdir {{remote_tmp_dir | expanduser }}/zypper2/rpm-build"
+    --define "_builddir %{_topdir}" \
+    --define "_rpmdir %{_topdir}" \
+    --define "_srcrpmdir %{_topdir}" \
+    --define "_specdir {{remote_tmp_dir | expanduser}}/zypper2" \
```

Comment:
... and this one could also be adjusted. Again, this is nit picking, so not holding the PR back because of it.